backend/moonstreamapi/admin/subscription_types.py

```diff
@@ -17,6 +17,26 @@
 from ..settings import bugout_client as bc
 
 CANONICAL_SUBSCRIPTION_TYPES = {
+    "ethereum_smartcontract": SubscriptionTypeResourceData(
+        id="ethereum_smartcontract",
+        name="Ethereum smartcontracts",
+        choices=["input:address", "tag:erc721"],
```

Comment:
Remove `"tag:erc721"`.

Reply:
@Andrei-Dolgolev will do this in a separate PR. https://github.com/bugout-dev/moonstream/issues/493